Merge back changes since April

martinwe-adfinis opened the pull request:
Time to keep things up to date!

ayekat commented on etc/sh/profile.d/99-private.sh:
```diff
@@ -0,0 +1,11 @@
+# Load private profile.d snippets
```
This used to be already done with `etc/sh/profile` (now moved into `etc/sh/profile.d/90-private.sh` in cea0a059e4f05ee47c53628ed8e8e219c605923b). Was that somehow missed?

Anyway, this seems unnecessary to me.

martinwe-adfinis replied:
Somehow missed that completely…. I've removed the commit.

ayekat commented:
On the scrlock change, I had already pointed this out in the previous MR (#39), but on second thought, I think it's fine to have this here for now until we've (slowly) cleaned up the packaging mess around karuiwm.

ayekat commented:
Works for me, merging.

martinwe-adfinis commented:
Rebased.

martinwe-adfinis committed:
etc/qutebrowser: Add Ubuntu package site as search engine

martinwe-adfinis committed:
etc: Stop ignoring gtk-3.0/bookmarks

I haven't seen any GTK bookmarks recently, but if they show up again,
they should probably be ignored on that specific level there, to avoid
confusion.

martinwe-adfinis committed:
etc/qutebrowser/config.py: Use shorthand config notation with `c.`

martinwe-adfinis committed:
etc/qutebrowser: Load session-specific configuration

Search in $LOCALDIR_PRIVATE/qutebrowser/sessions/$SESSION for
session-specific configuration. Only do so if there are actually
dwb-style "sessions" (i.e. basedir in $XDG_RUNTIME_DIR, etc.).

martinwe-adfinis committed:
Add etc/GNOME-xdg-terminals.list

No idea what this is, but I suspect it lists the available terminal
emulators for Gnome. I can live with the fact that it simply states
gnome-terminal (as I'm using that as my alternative/pretty/zoomable
one).
